Geometry/MTDNumberingBuilder/src/CmsMTDStringToEnum.cc

```diff
@@ -22,46 +22,13 @@ CmsMTDStringToEnum::Impl::Impl() {
                                                                                   GeometricTimingDet::ETL));
   _map.insert(
       std::pair<std::string, GeometricTimingDet::GeometricTimingEnumType>("Disc1", GeometricTimingDet::ETLDisc));
+  _map.insert(
```

Comment:
it would be nicer to use `emplace` here (could be addressed in a followup PR)

Reply:
@kpedro88 this is a quite old code, I'll address a few minor technical issues in a next PR, just to avoid keeping this one on hold for too long